src/main/kotlin/calculator/SplitString.kt

```diff
@@ -0,0 +1,40 @@
+package calculator
+
+import java.lang.NumberFormatException
+
+object SplitString {
+    private const val SPACE = " "
+
+    fun getSplitExpression(expression: String?): List<String> {
+        checkExpression(expression)
+        return expression!!.split(" ")
+    }
+
+    private fun checkExpression(expression: String?) {
+        checkNullOrBlankString(expression)
+        val splitExpression = expression!!.split(SPACE)
+        for (i in splitExpression.indices) {
```

Comment:
```suggestion
        for (i in splitExpression.indices step 2) {
```
이런식으로 step N 을 활용해서 2칸씩 옮길수도있어